GVFS/GVFS.Common/Maintenance/PostFetchStep.cs

```diff
@@ -48,12 +48,15 @@ protected override void PerformMaintenance()
 
                 this.RunGitCommand((process) => process.WriteCommitGraph(this.Context.Enlistment.GitObjectsRoot, this.packIndexes), nameof(GitProcess.WriteCommitGraph));
 
+                // Turning off Verify for commit graph due to performance issues.
+                /*
                 GitProcess.Result verifyResult = this.RunGitCommand((process) => process.VerifyCommitGraph(this.Context.Enlistment.GitObjectsRoot), nameof(GitProcess.VerifyCommitGraph));
 
                 if (verifyResult.ExitCodeIsFailure)

```

Comment:
@jeschu1 This `if` statement needs to be updated to check if we are stopping the step! The process can fail if we halt due to an unmount command.

Reply:
(I'll make a new PR that does that, and does the same for the multi-pack-index verify command.)